lib/tests/misc.nix

```diff
@@ -708,7 +740,7 @@ runTests {
       # should just return the initial value
       emptySet = foldlAttrs (throw "function not needed") 123 { };
       # should just evaluate to the last value
-      accNotNeeded = foldlAttrs (_acc: _name: v: v) (throw "accumulator not needed") { z = 3; a = 2; };
+      valuesNotNeeded = foldlAttrs (acc: _name: _v: acc) 3 { z = throw "value z not needed"; a = throw "value a not needed"; };
```

Comment:
`foldlAttrs`?
The name doesn't look like a strict function's name...
I guess we might still apply the same reasoning here, but I don't like that it isn't `foldl'Attrs`. I guess not a lot of people would use it in a lazy way, so I guess I won't make a fuss, but this change should be noted in the release notes too.

Reply:
Added a release note now!